licenses/LICENSE_nishtahir_language-kotlin

```diff
@@ -0,0 +1,173 @@
+LICENSE retrieved from https://raw.github.com/nishtahir/language-kotlin/0fa40c0/LICENSE
```

Comment:
From what I read, [linguist](https://github.com/github/linguist) uses a github action for [licensed](https://github.com/github/licensed) to bundle the licenses. But in this project licenses get handled via the `license_path` attribute of the `Repo` class and I don't know if there are any efforts to change this.

Reply:
I'm not making any recommendations on how to solve the problem since I don't really know this project or how it's run. I'm only commenting that I have a NOTICE file in my repo and if you are working with licenses on this project the author may want to consider pulling that somehow as part of the integration since my work builds on top of work done by others.